pkg/settings/settings.go

```diff
@@ -135,6 +135,12 @@ var (
 		editable,
 		initializeFrom("true"),
 		modifyWith(notBlank))
+	// SSLSkipVerify indicates whether skip SSL verification when accessing remote server.
+	SSLSkipVerify = newValue(
```

Comment:
it seems we are doing skip "insecure" tls cert verification, can we use something like`DisableInsecureVerify` or `SkipInsecureVerify`?

Reply:
updated